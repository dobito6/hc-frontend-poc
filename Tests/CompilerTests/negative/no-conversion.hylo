fun main() {
  _ = () as {Void, Void}
}
