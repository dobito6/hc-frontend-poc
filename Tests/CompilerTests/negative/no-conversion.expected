negative/no-conversion.hylo:2.7-9: error: no conversion from 'Void' to '{Void, Void}'
  _ = () as {Void, Void}
      ~~
